tools/afl-fuzzer/inputs/input1.txt

```diff
@@ -1 +0,0 @@
-100 4 aaaa
```

Comment:
Are these text and binary input files needed ?

Reply:
Not really. I just wanted to provide a sample files to let users know of the format of input files. I think i can include it in the main file itself as comments i.e the format.

Reply:
Anything dynamically generated should be left out in my opinion unless it is extremely hard to generate (for example initial IPC captures that are used to seed the fuzzer)